Simulations/python/run_recipes.py

```diff
@@ -52,12 +81,40 @@ def runRecipes(argv):
 
 
     # if simulations were done, update the headers
-    if(not simulationSet.params['testRun']):
+    if not simulationSet.params['testRun']:
         simulationSet.updateHeaders()
 
-    if(simulationSet.params['doStatic']):
+    if simulationSet.params['doStatic']:
         makeCalibPrototypes.generateStaticCalibs(simulationSet.params['outputDir'])
         
 if __name__ == "__main__":
+    simulationSet = rr.runRecipes()
+    simulationSet.parseCommandLine(sys.argv[1:])
```

Comment:
I don't understand, this doesn't seem to make sense: all the code is now duplicated and not in a separate function anymore. Neither of those is good; my suggestion to make a PR was exactly to prevent code duplication.

What I expected was that you would refactor the `runRecipes()` functions into two parts, so the code would look something like:

```python
def runRecipes_with_pars(
    inputYaml=...,
    outputDir=...,
    ...
):

def runRecipes_with_argv(argv):
    params = parseCommandLine(argv[1:])
    runRecipes_with_pars(**params)

if __name__ == "__main__":
    runRecipes_with_argv(sys.argv)
```
Or something equivalent. (These names are snakecamelcase...)